src/controllers/UndoController.ts

```diff
@@ -0,0 +1,37 @@
+import { EditorAPI } from '../../types/CommonTypes';
+
+/**
+ * The UndoController is responsible for all communication regarding the Undo-Manager.
+ * Methods inside this controller can be called by `window.SDK.undoManager.{method-name}`
+ */
+export class UndoController {
```

Comment:
I think we should ask this controller Name to the Product.It includes undo and redo but we only call it undoController.

Reply:
Kind of true, also on editor-engine it's called UndoService, so controller can have the same name (UndoServiceController).
On this.x level it should indeed be validated by product. SDK.undo.undo would be a bit weird.